### PR TITLE
FF132 Update third-party cookies article Firefox details (ETP Strict 3pc blocking)

### PR DESCRIPTION
**Pull request by leanderschwarz**
### Description

As of Firefox 132 (release October 29th) Enhanced Tracking Protection Strict mode will block all third-party cookies. Developers may still opt-into third-party cookies using CHIPS.

### Motivation

The Firefox Privacy Team wants to update all relevant documentation regarding third-party cookie blocking for the Firefox 132 release.

The article should reflect the most recent Firefox (third-party) cookie behavior. Developers should know that CHIPS can be used to opt-into partitioned third-party cookie usage with Enhanced Tracking Protection Strict third-party cookie blocking.

### Additional details
See https://bugzilla.mozilla.org/show_bug.cgi?id=1918037.

Since this is my private account I am adding my Mozilla mail in case further questions arise: 
lschwarz@mozilla.com

### CODE REVIEW

File: files/en-us/web/privacy/third-party_cookies/index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Legislation such as the [General Data Privacy Regulation](https://gdpr.eu/) (GDP
...
+- Mozilla's [Anti-tracking policy](https://wiki.mozilla.org/Security/Anti_tracking_policy) has led to Firefox blocking third-party cookies from known trackers by default (see [Firefox tracking protection](/en-US/docs/Web/Privacy/Firefox_tracking_protection) and [Enhanced tracking protection](https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop)). Firefox also gives third-party cookies a separate cookie jar per site, so they can't be used to track users across sites (see [Total Cookie Protection](https://blog.mozilla.org/en/products/firefox/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/)). In [Enhanced tracking protection](https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop) Strict mode Firefox now blocks all third-party cookies. Website developers may continue to use third-party cookies with separate cookie jar per site (partitioned cookies) through [Cookies Having Independent Partitioned State (CHIPS)](/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies).
```
Comment (chrisdavidmills) [inline files/en-us/web/privacy/third-party_cookies/index.md:69]:
```suggestion
- Mozilla's [Anti-tracking policy](https://wiki.mozilla.org/Security/Anti_tracking_policy) has led to Firefox blocking third-party cookies from known trackers by default (see [Firefox tracking protection](/en-US/docs/Web/Privacy/Firefox_tracking_protection) and [Enhanced tracking protection](https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop)). Firefox also gives third-party cookies a separate cookie jar per site, so they can't be used to track users across sites (see [Total Cookie Protection](https://blog.mozilla.org/en/products/firefox/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/)). In [Enhanced tracking protection](https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop) Strict mode, Firefox now blocks all third-party cookies. Website developers may continue to use third-party cookies with a separate cookie jar per site (partitioned cookies) via [Cookies Having Independent Partitioned State (CHIPS)](/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies).
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks for the update @leanderschwarz. I've made a couple of small grammatical suggestions, but nothing major.

Since this is an addition/change for Fx132:

- If it is turned on in release, you should also add an entry for it to the [132 rel notes page](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/132), probably under the "Security" heading?
- If it is only enabled in Nightly/beta for now, add a note under the "Experimental web features" section instead, and an entry on the separate [Experimental features in Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features) page.

To get an idea of the style of the entries see the previous rel note pages.

Also giving @Rumyra a ping about this one — are you tracking this for the 132 release project?

**Comment by leanderschwarz** [comment]
Thanks for the quick review and changes @chrisdavidmills, I commited your suggestions!

This will be turned on in release thanks for the suggestion, I already talked to @Rumyra and we will add a note to the developer release notes.

**Review by chrisdavidmills — APPROVED** [review]
@leanderschwarz perfect! Let's get this one merged.

**Pull request merged**